conf/master

```diff
@@ -294,7 +299,8 @@
 # buffering.
 # Value: In bytes. Set to 'dynamic' to have Salt select
 # a value for you. Default is disabled.
-# ipc_write_buffer: 'dynamic'
+#ipc_write_buffer: 'dynamic'
+#
```

Comment:
why is this added, it seems inconsistent.

Reply:
Because `dynamic` is **not** the default, hence shouldn't be followed by a blank line.